php-nginx/build-scripts/install_php70.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# A shell script for installing PHP 7.0.x.
+set -xe
+
+apt-get install -y \
+        gcp-php70 \
+        gcp-php70-apcu \
+        gcp-php70-apcu-bc \
+        gcp-php70-grpc \
+        gcp-php70-mailparse \
+        gcp-php70-memcached \
+        gcp-php70-mongodb \
+        gcp-php70-redis
+
+# Enable some extensions for backward compatibility
+${PHP70_DIR}/bin/php70-enmod apcu-bc
+${PHP70_DIR}/bin/php70-enmod apcu
+${PHP70_DIR}/bin/php70-enmod mailparse
+${PHP70_DIR}/bin/php70-enmod memcached
```

Comment:
Looking through the code, I understand it now. Essentially the container builds with php5.6 enabled, and then `composer.sh` gets executed to install the dependencies and change the php version if applicable.

Personally, I'd like to see a new script for enabling a PHP, i.e. `enable_php.sh`. And then remove `ln -s` from `install_php56.sh`. I understand right now we use `composer.json` alone to determine the version, but this might change. It could be split into `enable_php.sh` and `install_app_dependencies.sh`